CardiorespiratoryFitness/CardiorespiratoryFitness/Model/CRFTaskInfo.swift

```diff
@@ -135,8 +119,8 @@ public struct CRFTaskInfo : RSDTaskInfo, RSDEmbeddedIconVendor {
             return 2
         case .resting:
             return 1
-        case .stairStep:
-            return 5
+        case .heartSnapshot:
```

Comment:
It doesn't take 3 minutes. It takes 5.

Reply:
Well if you want to get technical it takes 4 minutes.  3 minutes for stair step, 1 minute for camera heart rate, I can switch it to that.

Reply:
1 minute to answer the demographics questions and *read* the instructions and results.

Reply:
That only applies to the first time you do the task though, every other run through you wouldn't read the instructions or do the question.  Seems like we should either go the extra mile and make it dynamic, or take the simple route and just round down to 4 minutes because it's more likely that is the true average length of time it takes to complete.

Reply:
Seriously? No. The whole point of estimating and giving an indication for how long it will take is for the first-time user. That's the whole point! Who wanted it changed? I can talk to them.

Reply:
I was told to make it 3 minutes, so I made it 3 minutes, that's really the extent of it.

Reply:
I will circle back with them and express how you feel about switching it to 5 minutes.

Reply:
And if you want to get technical about it, it takes ~1 second to start the sensors for the camera and 0.5 seconds to process the results. Each screen transition is on a 2 second animation and it is reasonable to assume that the user will take 0.5 seconds to respond to the screen change and tap the next button, therefore 2.5 seconds / screen.

There are 4 instruction screens = 10 seconds
There is a 10 second count down = 10 seconds + 2 second transition animation = 12 seconds
The stair step has a running time of 180 seconds = 182 seconds
The heart rate is 60 seconds + 2 second animation + 1 second to start the camera = 63 seconds
The feedback screen is 1 page. If they immediately close the page w/o reading it then that's 2.5 seconds.

Total = 269.5 seconds = 4.4916666667 minutes

It's always better to round up when estimating how long something is going to take so I don't think my 5 minutes is unreasonable.